src/server/pfs/cmds/cmds.go

```diff
@@ -310,6 +310,39 @@ $ pachctl list-commit foo master --from XXX
 	listCommit.Flags().IntVarP(&number, "number", "n", 0, "list only this many commits; if set to zero, list all commits")
 	rawFlag(listCommit)
 
+	printCommitIter := func(commitIter client.CommitInfoIterator) error {
+		if raw {
+			for {
+				commitInfo, err := commitIter.Next()
+				if err == io.EOF {
```

Comment:
Ahh, I see so, it does call `Flush` but you can only call `Flush` on a `TabWriter` once which should happen after the last call to `Write`. There's an inherent problem here which is that there is no last call to `Write` when there's a never ending stream of things to `Write`.